hatServer/static/js/services/surveyQuestions.js

```diff
@@ -0,0 +1,461 @@
+'use strict';
+
+sortingApp.service('surveyQuestions', function () {
+    var surveyName = "";
+    var surveyDescription = 'HELLO JESSICA YOU DA BEST';
+    var desiredSkills = ['Java', 'Python', 'PHP', 'C/C++', 'Mobile App Development (Android/iOS)', 'Web Applications', 'Embedded Systems', 'Database (MySQL, SQL, etc.)', 'User Interface/Experience', 'Statistics', 'Networking',
+        'Robotics', 'Computer Vision', 'Algorithms', 'Machine Learning'];
+    var projects = [];
+    var firstName = {
+        included: true,
+        required: true
+    };
+    var lastName = {
+        included: true,
+        required: true
+    };
+    var preferredName = {
+        included: true,
+        required: false
+    };
+    var overallGPA = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var csGPA = {
+        included: true,
+        required: true,
+        weight: 1
```

Comment:
Are these weights passed in the JSON object? I'm not sure I understand how they're used here, or do these values get input by the instructors when they determine how to weight variables? Either way, it'd probably be easier if the weight was passed in a separate interface and could just be loaded into the algorithm variable.cfg file